app/views/user_mailer/assessment_job_email.html.erb

```diff
@@ -7,15 +7,15 @@
 <br />
   Total eligible for notifications at runtime: <%= @eligible %>
 <br />
-Sent during this job run: <b><%= @sent.count %></b>
+Sent during this job run: <b><%= @sent.size %></b>
 <br />
-Not sent during this job run (due to exceptions): <b><%= @not_sent.count %></b>
+Not sent during this job run (due to exceptions): <b><%= @not_sent.size %></b>
 <br />
 <h2>
   Sent
 </h2>
 <br />
-ID, Method
+Method, Count
```

Comment:
is this not supposed to be 'ID, Method'?

Reply:
good catch. fixing now